tests/__init__.py

```diff
@@ -44,7 +45,22 @@ def is_gevent_monkey_patched():
     return socket.socket is gevent.socket.socket
 
 
+def is_gevent_time_monkey_patched():
+    import gevent.monkey
+    return "time" in gevent.monkey.saved
+
+
+def is_eventlet_time_monkey_patched():
+    import eventlet
+    return eventlet.patcher.is_monkey_patched('time')
+
+
 def is_monkey_patched():
     return is_gevent_monkey_patched() or is_eventlet_monkey_patched()
 
+
+MONKEY_PATCH_LOOP = bool(os.getenv('MONKEY_PATCH_LOOP', False) == "1")
```

Comment:
One last nit: it looks like other env vars used as flags just check for its presence in the environment, not its value, e.g. `USE_CASS_EXTERNAL = bool(os.getenv('USE_CASS_EXTERNAL', False))` -- so, this should just do `bool(os.getenv('MONKEY_PATCH_LOOP', False))`